app/src/main/java/com/eveningoutpost/dexdrip/webservices/WebServiceModule.java

```diff
@@ -19,69 +21,77 @@
 @Module
 public class WebServiceModule {
 
-    private static final boolean d = true;
+    private static final String TAG = "WebServiceModule-INJECT";
```

Comment:
Hopefully, this changed log entry name does not affect any downstream log analyses. Including the class name is indeed usefull.